src/sentry/tasks/integrations/migrate_opsgenie_plugin.py

```diff
@@ -82,13 +78,12 @@ def migrate_opsgenie_plugin(integration_id: int, organization_id: int) -> None:
             for rule in Rule.objects.filter(project_id=project.id)
             if ALERT_LEGACY_INTEGRATIONS in rule.data["actions"]
```

Comment:
Will this check for any item in the list containing the `id` of `"sentry.rules.actions.notify_event.NotifyEventAction"` or only an item that contains the exact keys and values. We want it to check the latter since the DB might have items that look like:
```json
{"id": "sentry.rules.actions.notify_event.NotifyEventAction"}
```
or
```json
{"id": "sentry.rules.actions.notify_event.NotifyEventAction", "name": "Send a notification to legacy integrations"}
```
We want to make sure`rules_to_migrate` has all rules with either of these blobs under `actions`.

This might would be a good check to add a test case for.